cogs/fun.py

```diff
@@ -117,31 +118,16 @@ async def joke(self, ctx: commands.Context[CodingBot]):
         ------
         `{prefix}joke`: *will get a random joke*
         """
-        joke_json = await self.http.api["joke"]["api"]()
+        joke_json = json.loads(await self.http.api["joke"]["api"]())
```

Comment:
Instead of converting text to json explicitly in commands, it will be better to bypass the `application/json` mime validation as a whole and let the error be handled at top level
Make changes here:
https://github.com/The-Coding-Realm/coding-bot-v6/blob/f108c90b7f29cf00efdec37a7cd8057250fdad21/ext/http.py#L169
Check [aiohttp docs for disabling mime validation](https://docs.aiohttp.org/en/stable/client_advanced.html#disabling-content-type-validation-for-json-responses)
And make changes to your commit